processor/src/test/java/org/mapstruct/ap/testutil/runner/CompilingStatement.java

```diff
@@ -102,12 +102,7 @@ protected CompilationCache initialValue() {
         }
     };
 
-    private static final List<String> LIBRARIES = Arrays.asList(
```

Comment:
Nice to get rid of the copying below. But should we establish a filter / white-list of the deps to propagate we actually want to avoid exposing too much?